composer.json

```diff
@@ -21,5 +21,10 @@
             "" : "src/"
         }
     },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "2.0.x-dev"
+        }
+    },
```

Comment:
after reading the [documentation](https://getcomposer.org/doc/articles/aliases.md#branch-alias), I understand that, now, if someone requires `2.0.*`, composer will install the `master` branch instead of installing a tagged version... tell me if I'm wrong